medusa/notifiers/kodi.py

```diff
@@ -281,7 +283,7 @@ def clean_library(self):
         # If no errors, return True. Otherwise keep sending command until all hosts are cleaned
         return clean_library
 
-    def _update_library(self, host=None, series_name=None):
+    def _update_library(self, host=None, series_name=None):  # pylint: disable=too-many-return-statements, too-many-branches
```

Comment:
Why the pylint?

Reply:
Not sure. Didn't put it in. Maybe because of a merge/rebase? I'll remove it